site/topics/booleans/booleans.rst

```diff
@@ -164,3 +164,91 @@ More about returns
     * Now write it with only **1** line of code within the function (so, 2 lines including the function header).
    
 * Functions returning Boolean values are pretty handy. Why? Where do you see yourself using them?
+
+
+Composing Operators and Values
+==============================
+
+* We have seen different operators that work on different types
+* As long as the types check out and we follow the `order of operations <https://docs.python.org/3/reference/expressions.html#operator-precedence>`_, we can compose rather complex expressions
+
+    * Don't worry about memorizing the order of operations
+    * They follow what you are used to for the most part
+    * When in doubt, make use of parentheses
+
+
+Evaluating Example Expressions
+------------------------------
+
+**``((17 + 2) < 18) or (17 != 18)``**
+
+``(19 < 18) or (17 != 18)``
+
+``False or (17 != 18)``
+
+``False or True``
+
+``True``
+
+
+**``(14 > 0) or ((6 != 7) and ((4 + 17) < 20))``**
+
+``True or ((6 != 7) and ((4 + 17) < 20))``
+
+``True or (True and ((4 + 17) < 20))``
+
+``True or (True and (21 < 20))``
+
+``True or (True and False)``
+
+``True or False``
+
+``True``
+
+* Notice that once we evaluated ``(14 > 0)`` as ``True``, we really didn't need to finish evaluating the remainder of the expression
+* This is because, as long as one of the operands for an ``or`` is ``True``, we know the whole expression is ``True``
+
+
+**``(101 == 100) and ((((66 - 17) > 18) or ((19 - 22) < 0)) and (((4 + 17) < 20) or ((66 - 17) < 44)))``**
```

Comment:
This seems needlessly complex. I don't think it's helping to show such a long expression being evaluated.